srcs/client/src/Components/Tools/Box.css

```diff
@@ -1,4 +1,51 @@
 
+/* BOOTSTRAP */
+.scroll
+{
+	display: flex;
+	flex-direction: column;
+	overflow-y: scroll;
+	-ms-overflow-style: none;
+	scrollbar-width: none;
+}
+
+.scroll::-webkit-scrollbar {
+	display: none;
+  }
+
+.boxBlue
```

Comment:
We are not consistent with our css naming conventions. We are using both camelCase and kebab-case in the project. At this stage this is not a big issue but may be confusing globally.

Reply:
true. I think front end switched to kebab case with the introduction of bootstrap